### PR TITLE
docs: elasticloadbalancingv2_target_group

### PR DESCRIPTION
**Pull request by onceupon**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates #469

### CODE REVIEW

**Comment by catlike** [comment]
LGTM

File: examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_instance.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  tags = [
+    {
+      key   = "Name"
+      value = "main"
+    },
+    {
+      key   = "Modified By"
+      value = "AWSCC"
+    }
+  ]
```
Comment (wellsiau-aws) [inline examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_instance.tf:23]: Keep the tags consistent with the rest of resources

```suggestion
  tags = [
    {
      key   = "Modified By"
      value = "AWSCC"
    }
  ]
```

File: examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_ip.tf [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  name     = "ip-example"
+  port     = 80
+  protocol = "HTTP"
+  target_type = "ip"
+  vpc_id   = awscc_ec2_vpc.main.id
+  tags = [{
+    key   = "Modified By"
+    value = "AWSCC"
+  }]
+}
```
Comment (wellsiau-aws) [inline examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_ip.tf:11]: run `terraform fmt` to fix indentation

File: examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_lambda.tf [hunk 3]
```diff
@@ -0,0 +1,8 @@
+resource "awscc_elasticloadbalancingv2_target_group" "lambda-example" {
+  name     = "lambda-example"
+  target_type = "lambda"
+  tags = [{
+    key   = "Modified By"
+    value = "AWSCC"
+  }]
+}
```
Comment (wellsiau-aws) [inline examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_lambda.tf:8]: same here, run `terraform fmt` to fix indentation

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
minor nit pick

File: examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_instance.tf (hunk 1 above)
Comment (onceupon) [reply examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_instance.tf:23]: i was using the `Name` tag to give a name to the VPC resource, which can only be given through tags, but it's alright for this example vpc to be nameless.

File: examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_lambda.tf (hunk 3 above)
Comment (onceupon) [reply examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_lambda.tf]: I have rerun `terraform fmt` on the resources file, thanks for noticing.

File: examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_ip.tf (hunk 2 above)
Comment (onceupon) [reply examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_ip.tf]: I have rerun `terraform fmt` on the resources file, thanks for noticing.

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thanks!!

**Comment by marcosentino** [comment]
@onceupon Thank you for the contribution!

**Pull request merged**